module4/pd/lessons/linkedin_challenge.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: LinkedIn Challenge
+module: 4
+---
+
+## CareerFlow Competition!
+
+### Instructions
+
+* Install the [Careerflow Chrome Extension](https://chromewebstore.google.com/detail/careerflow-ai-job-applica/iadokddofjgcgjpjlfhngclhpmaelnli?pli=1)
+* Go to your Linkedin profile and click the blue box with a "C" in it on the right-hand side of the page
+* Make the recommended changes to your profile to improve it - you have until 2PM MT to get your scores in
+* Tally up the scores of each team member and average them.
+
+The team spokesperson will DM your instructor(s) with your individual AND group's average score. **Scores must be submitted by 2PM MST!**  
+
+The team with the highest average overall score will be the winners! 🎉
+
+We will be creating a social media spotlight for the winning team members.  🔦
```

Comment:
Maybe we should remove the team phrasing here. The teams historically have been houses which barely exist by mod 4.

Reply:
I'm going to go ahead and update verbiage here on the branch.